libs/agno/agno/models/base.py

```diff
@@ -1223,6 +1223,9 @@ def run_function_call(
                         if function_call.function.show_result:
                             yield ModelResponse(content=item.content)
 
+                    elif isinstance(item, CustomEvent):
+                        function_call_output += str(item)
```

Comment:
Ooh should we do this? Is this needed? The idea wasn't really that it becomes part of the output of the function itself, but rather that it is just an event...
But I don't really see another way, so probably this is right, because you can't yield and return something